src/jsonobject.ts

```diff
@@ -290,23 +290,27 @@ export class JsonObjectProperty implements IObject {
     return this.onGetValue || this.serializationProperty;
   }
   public get defaultValue() {
-    var result: any = this.defaultValueValue;
+    let result: any = this.defaultValueValue;
     if (
       !!JsonObjectProperty.getItemValuesDefaultValue &&
       JsonObject.metaData.isDescendantOf(this.className, "itemvalue")
     ) {
-      result = JsonObjectProperty.getItemValuesDefaultValue(
-        this.defaultValueValue || []
-      );
+      result = JsonObjectProperty.getItemValuesDefaultValue(this.defaultValueValue || [], this.className);
     }
     return result;
   }
   public set defaultValue(newValue) {
     this.defaultValueValue = newValue;
   }
   public isDefaultValue(value: any): boolean {
-    if (!Helpers.isValueEmpty(this.defaultValue))
+    if (!Helpers.isValueEmpty(this.defaultValue)) {
```

Comment:
The following code will work:
    if (!Helpers.isValueEmpty(this.defaultValue))
      return Helpers.isTwoValueEquals(value, this.defaultValue, false, true, false);

Reply:
Fixed! Btw, you can actually suggest a code change via the Github UI!